packages/core/package.json

```diff
@@ -23,12 +23,16 @@
   },
   "//": "@types/cls-hooked is exposed in API so must be in dependencies, not devDependencies",
   "dependencies": {
-    "@aws-sdk/types": "^3.4.1",
     "@aws-sdk/service-error-classification": "^3.4.1",
+    "@aws-sdk/types": "^3.4.1",
     "@types/cls-hooked": "^4.3.3",
+    "@types/node-fetch": "^2.6.3",
     "atomic-batcher": "^1.0.2",
     "cls-hooked": "^4.2.2",
-    "semver": "^7.3.8"
+    "mocha": "^10.2.0",
+    "semver": "^5.3.0",
+    "tsd": "^0.28.1",
+    "typescript": "^5.0.4"
```

Comment:
mocha, tsd, and typescript are not needed here since they get pulled from the [root package.json](https://github.com/aws/aws-xray-sdk-node/blob/master/package.json#L44C24-L52)